src/main/java/core/basesyntax/service/impl/ReportGeneratorImpl.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.dao.FruitDao;
+import core.basesyntax.service.ReportGenerator;
+import java.util.stream.Collectors;
+
+public class ReportGeneratorImpl implements ReportGenerator {
+    private static final String reportHead = "fruit,quantity";
```

Comment:
The constant 'reportHead' should follow Java naming conventions for constants with all letters being uppercase and words separated by underscores. For example, 'REPORT_HEAD'.

Reply:
++